server/api-manager.js

```diff
@@ -943,14 +939,17 @@ const MindbodyAccess = new MindbodyQueries();
 export default MindbodyAccess;
 
 /*example code
-var client = new mindbodyQueries();
-client.getAuth()
-  .getClients()
-  .then((value) => {
-    console.log(value);
+import MindbodyAccess from "../../api-manager.js";
```

Comment:
I love getting this in. It's a shame that we duplicated this work so many times. There's probably a better way to use git that would have made this happen faster and more efficiently. Maybe we could discuss in sprint retrospective or some other meeting.
Anyway, all's well that ends well.